development.postgresql.yml

```diff
@@ -40,6 +40,7 @@ services:
       - mdb 
     volumes:
       - ./notarysql/postgresql-initdb.d:/docker-entrypoint-initdb.d
+    command: -l
```

Comment:
for my own understanding: where does this come from?  I didn't find this option on the official image description...tried searching around but couldn't find the flag elsewhere